CHANGELOG.rst

```diff
@@ -1,5 +1,73 @@
 .. _changelog:
 
+0.7.3
+-----
+
+This release contains some very nice improvements to the file and datasets management and visualization as well as project migrations.
+
+New features
+~~~~~~~~~~~~
+
+- **Dataset removal** A dataset can be now removed from a project from the UI. Note that if a removed dataset was part of a project with the Knowledge Graph activated, its metadata will not be removed.
+
+- **Pinned environments image** An image tag can be specified for launching environments. This can be done in the `renku configuration file <https://renku.readthedocs.io/en/latest/user/templates.html?highlight=.dockerignore#renku>`_.
```

Comment:
I suggest changing the wording:
```
A project can pin interactive environments to a specific image, independent of the content of the project. This can be useful for situations like courses where everyone should use the environment defined by the instructor.

To take advantage, see the `renku configuration file documentation <https://renku.readthedocs.io/en/latest/user/templates.html?highlight=.dockerignore#renku>`_.
```